src/loaders/compute_tools/checkm2/checkm2.py

```diff
@@ -68,6 +69,13 @@ def _run_checkm2(
         fatal_tuples.append(fatal_tuple)
     write_fatal_tuples_to_dict(fatal_tuples, output_dir)
 
+    metadata = {'tool': 'checkm2',
+                'version': '1.0.1',
+                'command': command,
+                'reference_db': 'diamond_db ver unknown - saved copy in NERSC at global/cfs/cdirs/kbase/collections/libraries/checkm2/1.0.1/uniref100.KO.1.dmnd',
+                'ids_to_files': ids_to_files}
```

Comment:
Remind me what ids_to_files is again?

Reply:
that's `data_id: source_file_path`.

Reply:
Why is that valuable to put in the meta? I'm not saying it isn't, I'm just not quite sure.

Reply:
I want to use the `data_id` as a foreign key in CDM. We can still parse it from tool results. But this is better and direct I think.

Reply:
So you can look up the tool run from the genome id?

Reply:
We can. But I was thinking to join tool result tables with downloaded metadata tables.

Reply:
Won't that duplicate a ton of data since it's the same for every data id?

Reply:
no. it won't. Only batched tool has this. So it will be one per batch.

Reply:
I don't understand what you were saying then. Are you talking about having some sort of run iD that's associated with each row of the tool results that's linked to a single row of tool metadata in another table?

Reply:
e.g join eggnog result to checkm2 results; join exiting metadata (the csv files we imported already) to tool results.

Reply:
By join do you mean a SQL JOIN or are you talking about adding more rows to a csv file?

Reply:
SQL join. We am planning to create table for each tool. e.g. fastgenomics.eggnog_result; fastgenomics.checkm2_results. And using this data_id as foreign.

Reply:
Ok, you might want to think abuot a run id since you can run the same tool multiple times on the same data_id, but :+1: